src/wasm/wat-parser.cpp

```diff
@@ -288,20 +288,27 @@ struct ImportNames {
 // Parser Contexts
 // ===============
 
+using IndexMap = std::unordered_map<Name, Index>;
```

Comment:
I would prefer to keep this as-is since it is slightly less verbose and we'll have a lot of `IndexMap`s eventually. It will be clear at all their use sites that the keys are names.